src/tadam.jl

```diff
@@ -0,0 +1,321 @@
+export tadam, TadamSolver
+
+"""
+    tadam(nlp; kwargs...)
+
+Trust-region embeded ADAM (TADAM) algorithm for unconstrained optimization. This is an adaptation of ADAM which enforces convergence in the non-convexe case.
+
+# Minimal algorithm description 
+The step sk at iteration k is computed as:
```

Comment:
Please apply all the same kinds of comments as for FOMO related to spacing, blank lines, indentation, etc.